src/components/cylc/Header.vue

```diff
@@ -86,24 +91,18 @@ const store = useStore()
 // owner logic
 const ownerOnLoad = store.state.user.user.owner
 const owner = ref(ownerOnLoad)
-const owners = ref([ownerOnLoad])
-const addOwner = (owner) => {
-  if (owners.value.indexOf(owner) === -1) { owners.value.push(owner) }
-}
+const owners = ref(new Set([ownerOnLoad]))
 
```

Comment:
[Sets](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) are a useful container for when you don't want duplicate items, both in JS and Python